api/v1/orcid/OrcidController.php

```diff
@@ -168,9 +169,12 @@ function (Role $role) {
             }
         }
 
+        $clonedAuthor = clone $author;
         $author->setOrcid(null);
         $author->setOrcidVerified(false);
+
         OrcidManager::removeOrcidAccessToken($author);
+        dispatch(new RevokeOrcidToken($context, $clonedAuthor));
```

Comment:
You could dispatch the job within `OrcidManager::removeOrcidAccessToken()`, that way you wouldn't have to duplicate/remember to additionally dispatch the job and you wouldn't need to clone the author/user either.

Reply:
good suggestion, I've made the update.